restful-api.md

```diff
@@ -2,11 +2,9 @@
 # RESTful API for Bitkub (2023-04-19)
 
 # Announcement
-* Temporary Suspension of ```All Digital Currencies Deposit, Withdrawal, Trading, and All Activities``` on Public API for Scheduled System Improvement
-  * Scheduled System Improvement: ```31 May 2023, 12:01 AM - 06:00 AM (GMT+7)```
-
 
 # Change log
+* 2023-11-14 Update [Rate-limits](#rate-limits) of V2 endpoints and Update rate limit from per IP to per user.
```

Comment:
2023-11-14 Changed [Rate-limits](#rate-limits) counting of APIs V2 from per `IP` to per `user`.